docs/reference/commandline/trust_key_load.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "key-load"
+description: "The key-load command description and usage"
+keywords: "Key, notary, trust"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# trust key-load
+
+```markdown
+Usage:  docker trust key-load KEY [KEY...]
+
+Load a signing key
+
+```
+
+## Description
+
+Docker trust key-load adds keys to the local docker trust keystore
+
+## Examples
+
+### Sign a single key
+
+For a private key `alice.pem` with permissions `-rw-------`
+
+```bash
+$ docker trust key-load alice.pem
+
+Loading key from "alice.pem"...
+Enter passphrase for new signer key with ID f8097df: 
+Repeat passphrase for new signer key with ID f8097df: 
+Successfully imported key from alice.pem
+
+```
+`docker trust key-load` will load keys on a best effort basis, so it will contnue to load keys if one attempt fails:
```

Comment:
Taking from https://github.com/docker/cli/pull/472#pullrequestreview-58723787, we should probably consider rephrasing this with `s/will load/loads` and so on.

Reply:
Also, typo: `s/contnue/continue`

Reply:
We would need to update signer-add and signer-remove docs to remove the will as well then

Reply:
that's fine - we can do that here or in a separate PR